### ISSUE TITLE
Is there any method related to model selection？

### ISSUE THREAD

**Issue by wyx1010120806** [Zhuang-Zhuang-Liu/DeepUplift#2]
Now we have a training set, a validation set and a test set. We will use this test set to evaluate the effect of different uplift models. There may be chance, but the training cost is too high when using cross validation.I would like to ask if there is a better way to evaluate the effect of different uplift models on the current data set?
